ReactAndroid/src/main/java/com/facebook/react/modules/clipboard/ClipboardModule.java

```diff
@@ -65,6 +70,9 @@ public void getString(Callback cb) {
       }
     } catch(Exception e) {
       FLog.w(ReactConstants.TAG, "Cannot get clipboard contents: " + e.getMessage());
+      if(errorCb!=null){
```

Comment:
space between if and (, space around != and between ) and {, i.e.

    if (errorCb != null) {